src/common/snippets/src/lowered/pass/load_movebroadcast_to_broadcastload.cpp

```diff
@@ -27,7 +26,7 @@ bool LoadMoveBroadcastToBroadcastLoad::run(LinearIR& linear_ir) {
             const auto& interm_connector = expr->get_input_port_connector(0);
             const auto parent_expr = interm_connector->get_source().get_expr();
             const auto load = ov::as_type_ptr<op::Load>(parent_expr->get_node());
-            if (!load)
+            if (!load || expr->get_loop_ids() != parent_expr->get_loop_ids())
```

Comment:
Why do we change this transformation behavior in this PR? It there a reported bug or undesired behavior?
It is not clear why these changes are valid straight away.